web/src/main/java/cz/muni/fi/pa165/controllers/BottleController.java

```diff
@@ -22,12 +23,20 @@
 
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public final List<BottleDTO> getBottles() {
-        return bottleFacade.findAll();
+        List<BottleDTO> result = bottleFacade.findAll();
+        if(result == null) {
+            result = Collections.emptyList();
+        }
```

Comment:
Myslim, ze je lepsi poslat prazdne pole nez null. I prohlizec to lip zpracuje.. :D